src/_pytask/parameters.py

```diff
@@ -44,6 +45,16 @@
 """click.Option: An option to control pytask's verbosity."""
 
 
+_SORT_TABLE_OPTION = click.Option(
```

Comment:
Do we need a cli option or do we want to assume this is always configured via the configuration file.

Reply:
Very good point. I think we could actually drop the cli option and only support it via the config file.